### PR TITLE
Update/cli docs

### PR DESCRIPTION
**Pull request by kellyjandrews**
## Description

Vonage CLI is going GA - these updates look to modify the current state of the docs using all the new syntax.

## Deploy Notes

Just markdown changes

### CODE REVIEW

File: _documentation/en/number-insight/guides/number-insight-via-cli.md [hunk 1]
```diff
@@ -99,101 +56,89 @@ The Standard and Advanced Number Insight APIs provide even more information abou
...
 
-To use the Advanced API, use `insight:advanced` (or `ia`):
 
 ```bash
-$ nexmo insight:advanced --verbose 447700900000
+$ vonage numberinsights --level=advanced 15555555555
 ```
 
-Look for the following additional fields in the response:
+A typical response from the Advanced API looks like this:
```
Comment (superchilled) [inline _documentation/en/number-insight/guides/number-insight-via-cli.md:107]: As per standard, could mention that there will be a prompt:

```
This operation will charge your account. Proceed?
```

File: _documentation/en/number-insight/overview.md [hunk 2]
```diff
@@ -63,71 +63,46 @@ This example shows you how to use the [Nexmo CLI](/tools) to access the Number I
...
 
 > Note: Depending on your user permissions, you might need to prefix the above command with `sudo`.
 
-Use your `VONAGE_API_KEY` and `VONAGE_API_SECRET` from the [dashboard getting started page](https://dashboard.nexmo.com/getting-started-guide) to set up the Nexmo CLI with your credentials:
+Use your `VONAGE_API_KEY` and `VONAGE_API_SECRET` from the [dashboard getting started page](https://dashboard.nexmo.com/getting-started-guide) to set up the Vonage CLI with your credentials:
 
 ```
-$ nexmo setup VONAGE_API_KEY VONAGE_API_SECRET
+$ vinage config:set --apiKey=VONAGE_API_KEY --apiSecret=VONAGE_API_SECRET
 ```
```
Comment (superchilled) [inline _documentation/en/number-insight/overview.md:79]: Typo: vinage

File: _documentation/en/numbers/code-snippets/buy.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ You need to purchase a Vonage [virtual number](/concepts/guides/glossary#virtual
 
 This page shows you how to buy a number programmatically.
 
-> You can also cancel a number online, using the [developer dashboard](https://dashboard.nexmo.com/your-numbers) or from the command line, using the [Nexmo CLI](https://github.com/Nexmo/nexmo-cli#buying-a-number).
+> You can also cancel a number online, using the [developer dashboard](https://dashboard.nexmo.com/your-numbers) or from the command line, using the [Vonage CLI](https://github.com/vonage/vonage-cli#buying-a-number).
```
Comment (superchilled) [inline _documentation/en/numbers/code-snippets/buy.md:17]: `#buying-a-number` fragment identifier is unnecessary.

**Review by superchilled — CHANGES_REQUESTED** [review]

File: _documentation/en/numbers/code-snippets/cancel.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ If you no longer require a Vonage virtual number that you have [purchased]() you
 
 This page shows you how to cancel a number programmatically.
 
-> You can also cancel a number online, using the [developer dashboard](https://dashboard.nexmo.com/your-numbers) or from the command line, using the [Nexmo CLI](https://github.com/Nexmo/nexmo-cli#cancelling-a-number).
+> You can also cancel a number online, using the [developer dashboard](https://dashboard.nexmo.com/your-numbers) or from the command line, using the [Vonage CLI](https://github.com/vonage/vonage-cli#cancelling-a-number).
 
 Replace the following variables in the sample code with your own values:
```
Comment (superchilled) [inline _documentation/en/numbers/code-snippets/cancel.md:15]: #cancelling-a-number fragment identifier is unnecessary.

File: _documentation/en/application/vonage-cli.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+
+### Script mode
+
+In any environment where the Vonage CLI is installed, you can also use it in a script.  The only required argument is the application `name`.
+
+
+```shell
+vonage apps:create "Test Application 1" --capabilities=voice,rtc --voice_event_url=http://example.com/webhooks/event --voice_answer_url=http://example.com/webhooks/answer --rtc_event_url=http://example.com/webhooks/rtcevent
+```
+
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply _documentation/en/application/vonage-cli.md:109]: replace with: 
```
vonage apps:create "Test Application 1" --voice_event_url=http://example.com/webhooks/event --voice_answer_url=http://example.com/webhooks/answer --rtc_event_url=http://example.com/webhooks/rtcevent
```

File: _documentation/en/application/vonage-cli.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+```shell
+openssl rsa -in private.key -pubout -outform PEM -out public.key.pub
+```
+
+This generates `public.key.pub`. This is the public key you use in creating or updating your Vonage application:
+
+```shell
+vonage apps:create "Application with Public Key" --public_keyfile=public.key.pub
+```
+
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply _documentation/en/application/vonage-cli.md:137]: It does.

File: _documentation/en/number-insight/guides/number-insight-via-cli.md [hunk 7]
```diff
@@ -99,101 +56,89 @@ The Standard and Advanced Number Insight APIs provide even more information abou
 
 ### Using the Number Insight Standard API
 
-To use the Number Insight Standard API, use the `nexmo insight:standard` command
+To use the Number Insight Standard API, use the following command:
 
 ```bash
-$ nexmo insight:standard --verbose 447700900000
+$ vonage numberinsights --level=standard 15555555555
 ```
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply _documentation/en/number-insight/guides/number-insight-via-cli.md:64]: Yes - sorry :(

File: _documentation/en/number-insight/guides/number-insight-via-cli.md [hunk 8]
```diff
@@ -99,101 +56,89 @@ The Standard and Advanced Number Insight APIs provide even more information abou
...
 
 A typical response from the Standard API looks like this:
 
 ```
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply _documentation/en/number-insight/guides/number-insight-via-cli.md:67]: Yes - let's do that as well.

File: _documentation/en/number-insight/guides/number-insight-via-cli.md (hunk 1 above)
Comment (kellyjandrews) [reply _documentation/en/number-insight/guides/number-insight-via-cli.md:107]: same

File: _documentation/en/numbers/code-snippets/buy.md (hunk 3 above)
Comment (kellyjandrews) [reply _documentation/en/numbers/code-snippets/buy.md:17]: I didn't make that discretion - just replacing what's there

File: _documentation/en/numbers/code-snippets/cancel.md (hunk 4 above)
Comment (kellyjandrews) [reply _documentation/en/numbers/code-snippets/cancel.md:15]: see other comment

File: _documentation/en/application/overview.md [hunk 9]
```diff
@@ -140,14 +140,14 @@ There is further explanation of webhook timeouts in the [webhook documentation](
...
+2. Using the [Vonage CLI](/application/vonage-cli).
 3. Using the [Application API](/api/application.v2).
 4. Using one of the Vonage [Server SDKs](/tools).
 
 ## Managing applications using the CLI
 
-* [Managing applications using the Nexmo CLI](/application/nexmo-cli)
+* [Managing applications using the Vonage CLI](/application/vonage-cli)
 
 ## Code Snippets
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply _documentation/en/application/overview.md:153]: Please double-check and correct this if needed.

File: _documentation/en/application/vonage-cli.md [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+## Listing your applications
+
+To list your current applications use:
+
+```shell
+vonage apps
+```
+
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply _documentation/en/application/vonage-cli.md:41]: Good point - would you be able to write that section?

File: _documentation/en/application/vonage-cli.md [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+This displays a list showing the application ID, name and capabilities.
+
+## Show application details
+
+To show the details of a specific application (where the `APP_ID` is the application  ID of an application that already exists):
+
+```shell
+vonage app:show APP_ID
+```
+
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply _documentation/en/application/vonage-cli.md:51]: yes - please update

File: _documentation/en/client-sdk/overview.md [hunk 12]
```diff
@@ -92,6 +92,6 @@ product: conversation
 ## References
 
 * [Conversation API](/conversation/overview)
-* [Nexmo CLI](https://github.com/nexmo/nexmo-cli/tree/beta)
+* [Vonage CLI](https://github.com/vonage/vonage-cli/tree/beta)
 * [Tutorials](/client-sdk/tutorials)
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply _documentation/en/client-sdk/overview.md:96]: Correct

File: _documentation/en/client-sdk/setup/create-your-application.md [hunk 13]
```diff
@@ -15,32 +15,30 @@ In order to use the Client SDK, there are three things you need to set up before
...
-* Install the Nexmo CLI.
+* Install the Vonage CLI.
 
-To install the Nexmo CLI, run the following command in a terminal:
+To install the Vonage CLI, run the following command in a terminal:
 
 ```bash
-npm install -g nexmo-cli@beta
+npm install -g @vonage/cli @vonage/cli-plugin-conversations
 ```
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply _documentation/en/client-sdk/setup/create-your-application.md:32]: I think I intended it at first - but now I hate it.

File: _documentation/en/client-sdk/setup/create-your-application.md [hunk 14]
```diff
@@ -68,13 +66,13 @@ Create a User who will log in to Vonage Client and participate in the SDK functi
 Run the following command in your terminal to create a user named Alice: 
 
 ```bash
-nexmo user:create name="Alice"
+vonage apps:users:create Alice
 ```
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply _documentation/en/client-sdk/setup/create-your-application.md:70]: Users will soon be a core plugin and not need installation - it won't be tomorrow, but I didn't want to update the docs twice.

File: _documentation/en/number-insight/guides/number-insight-via-cli.md [hunk 15]
```diff
@@ -1,95 +1,52 @@
...
-United Kingdom
-
-[country_prefix]
-44
-```
-
-This human-readable output mirrors the field names and data available in the JSON response: it returns data about your request (`status`, `status_message`, `request_id`), details of the country that the number belongs to (`country_name`, `country_prefix` etc.) and how to format the number appropriately for that country (`national_format_number`).
-
-> If you do not see a response similar to that shown above, check your API credentials and ensure that you have installed Node.js and `nexmo-cli` properly.
+> If you do not see a response similar to that shown above, check your API credentials and ensure that you have installed Node.js and the Vonage CLI properly.
```
[the remark this answers is not in the document]
Comment (kellyjandrews) [reply _documentation/en/number-insight/guides/number-insight-via-cli.md:50]: If you think it would help - add it in.

**Review by superchilled — APPROVED** [review]

**Pull request merged**